apps/mobile/src/app/pages/page-signin/sign-in.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react-native';
+
+import SignIn from './sign-in';
```

Comment:
- [ ] - change name of sign-in.tsx to file to index.tsx then import will be shorter